conf.py

```diff
@@ -15,20 +35,26 @@
     'sphinx.ext.viewcode',
 ]
 
-intersphinx_mapping = {'python': ('https://docs.python.org/3.4', None),'BusDevice': ('https://circuitpython.readthedocs.io/projects/bus_device/en/latest/', None),'CircuitPython': ('https://circuitpython.readthedocs.io/en/latest/', None)}
-
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
 source_suffix = '.rst'
 
+# The encoding of source files.
+#
+# source_encoding = 'utf-8-sig'
+
 # The master toctree document.
 master_doc = 'README'
 
 # General information about the project.
 project = u'Adafruit BNO055 Library'
-copyright = u'2017 Radomir Dopieralski'
-author = u'Scott Shawcroft'
+copyright = u'2017, Radomir Dopieralski and Adafruit Industries'
+author = u'Radomir Dopieralski'
```

Comment:
I think this is the only change you actually need from this file. The rest are comments and the intersphinx mapping actually gets reduced.